packages/itwinui-css/src/bridge.scss

```diff
@@ -225,6 +246,22 @@
     --_iui-expandable-block-header-caption-color-hover: var(--iui-color-text-muted);
   }
 
+  // StrataKit AccordionItem border is muted
+  .iui-expandable-block {
+    --_iui-expandable-block-border-color: var(--ids-color-border-neutral-muted);
+  }
+
+  // StrataKit AccordionItem button background color is transparent
+  .iui-expandable-header {
+    --_iui-expandable-block-background-color: transparent;
+    --_iui-expandable-block-header-background-color-hover: var(--_iui-expandable-block-background-color);
+  }
+
+  // StrataKit AccordionItem content background color is transparent
+  .iui-expandable-content {
+    --_iui-expandable-block-background-color: transparent;
+  }
```

Comment:
Let's remove these expandable-block overrides. StrataKit only supports the "ghost"/"borderless" variant right now, but that doesn't mean we need to force ExpandableBlock to always be borderless.